components/layout_2020/geom.rs

```diff
@@ -368,6 +368,15 @@ impl<T: Copy> LogicalSides<T> {
     }
 }
 
+impl<T: Copy> From<T> for LogicalVec2<T> {
+    fn from(value: T) -> Self {
+        Self {
+            inline: value,
+            block: value,
+        }
+    }
+}
```

Comment:
Rather than putting this in the middle of `LogicalSides` implementations, it seems better around https://github.com/servo/servo/blob/fdaf44bbc0e12703490ff4f621ca983b14859418/components/layout_2020/geom.rs#L538-L554